### ISSUE TITLE
Customizable Buzzer/Cap mount

### ISSUE THREAD

**Issue by stylesuxx** [rgbskills/fpv_open_racer#11]
I think it would be cool to have a customizable Buzzer/Cap mount, just in case if someone has problem with the hole sizes. I can whip something up in openSCAD that is similar to your design and put it up on thingiverse where one can use the customizer.

Maybe it would also be a good idea to link matching (that will fit the default holes) buzzer and cap in the BOM.

**Comment by rgbskills** [comment]
Sounds good, I measured most of the caps I had in the house and they all seem to be the same diameter +/- 0.2 so it should be fine, the buzzers I have are smaller, https://www.banggood.com/5V-Buzzer-Alarm-Beeper-With-Cable-for-Eachine-QX70-QX90-QX95-NAZE32-F3-DIY-Micro-Brushed-FPV-Racer-p-1100338.html

This is cheap (I bought it for 0.99 it's now 2.15 for some reason) , and a full-size buzzer is gigantic compared to this. and most racers use this one. I tried fitting a full-sized one and there is just not enough space.


a better offer can always be found on banggood: 
5: https://www.banggood.com/5-PCS-Super-Loud-5V-Active-Alarm-Buzzer-Beeper-Tracker-95_5mm-for-Racing-Drone-p-1117207.html
10: https://www.banggood.com/10Pcs-95_5mm-RJX-5V-FPV-Racing-Drone-Buzzer-for-RC-Model-p-1583530.html?rmmds=detail-left-hotproducts__5&ID=6278523&cur_warehouse=CN
